Scripts/Core/EditorVR.cs

```diff
@@ -48,8 +46,13 @@ sealed partial class EditorVR : MonoBehaviour
 		DirectSelection m_DirectSelection;
 		Menus m_Menus;
 		UI m_UI;
+		Interfaces m_Interfaces;
+		MiniWorlds m_MiniWorlds;
```

Comment:
Yes. I misunderstood. However, m_MiniWorlds remains to be necessary because it and m_MultipleRayInputModule are used every frame in ProcessInput.

I have pushed changes that address this feedback. Would you like me to remove more of the references I added to evr.m_SomeModule? I figured that as long as we had them we might as well use them.